src/neurostatslib/basis.py

```diff
@@ -374,7 +375,7 @@ def _evaluate(self, *xi: NDArray) -> NDArray:
         Returns
         -------
         :
-            The basis function evaluated at the samples (number of samples x number of basis)
+            The basis function evaluated at the samples (number of samples x n_basis_funcs)
```

Comment:
```suggestion
            The basis function evaluated at the samples, shape (n_samples, n_basis_funcs)
```
To  be consistent with other dosctrings.